docs/DeveloperGuide.md

```diff
@@ -243,6 +243,8 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`    | expert user                                | view all archived clients      | see the clients that I am no longer working with                       |
 | `* *`    | expert user                                | unarchive clients              | display them when I start working with them again                      |
 | `* *`    | regular user                               | clear all clients              | delete irrelevant client data when I move to a new company             |
+| `* *`    | regular user                               | exit the app                   | start to relax                                                         | 
+
```

Comment:
I'm not sure if exiting the app is a user story, because I think it is written from a point of view that we are trying to solve their problems, and they may not be thinking of an app/system/software as a solution

Reply:
Hmmm ok I thought all the use cases should be a user story.